### ISSUE TITLE
UnwrapCL.exe has stopped working

### ISSUE THREAD

**Issue by Henry00IS** [sabresaurus/SabreCSG#104]
This happens a lot when "Generate Lightmap UVs" is enabled:

![UnwrapCL.exe has stopped working](https://user-images.githubusercontent.com/7905726/39661343-3420d406-5050-11e8-90d1-b4864dffa18a.png)

**Comment by Henry00IS** [comment]
Finally I found a way to consistently replicate this bug so we can research it. Simply place a 2DSE brush onto a box and UnwrapCL.exe will crash. In my game stair cases and vertex edited cubes crash it as well so it's not directly related to 2DSE.

![image](https://user-images.githubusercontent.com/7905726/39665643-df563c52-5097-11e8-82a2-63a7bddbc7e0.png)

**Comment by Henry00IS** [comment]
For anyone reading these posts through E-mail I significantly edited my post with new details. Even just having two NoCSG brushes can crash UnwrapCL.exe.

**Comment by Henry00IS** [comment]
I did a lot of research on the internet. First of all there's no known fix for this issue that works for us. I did notice Unity uses Blender through the command line to import .blend files (it passes a python script that creates a .fbx file that it can read).

We can use Blender to create lightmap UV2:
* Export the CSGModel to a .obj file.
* Execute Blender with a Python script that can execute:
* * "Lightmap Pack" or
* * "Smart UV Project"
* Export the result to a .obj file.
* Read the .obj file back into Unity.
* Parse the UV2 coordinates.

It's a neat workaround that no longer requires UnwrapCL.exe and should still be pretty fast although you wouldn't want to do this with Auto Rebuild. It would be another good opportunity to add a "Final Rebuild" button. Users could decide between using UnwrapCL.exe or Blender.exe in the SabreCSG preferences.
